### PR TITLE
Add "Contributing to the Handbook" page

### PR DESCRIPTION
**Pull request by zuren**
This guide aims to make it easier to submit changes to the Handbook without using `git` or a text editor.

### CODE REVIEW

File: guides/contributing_to_the_handbook.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+1. Click on the "Files changed" tab on your Pull Request.
+2. Click on the pencil icon in the toolbar above the file content.
+3. Make your changes as before, previewing until you're happy with them.
+4. Under "Commit Changes" describe what this change does. For example, "Fix typo" or "Extend section on getting a company credit card".
+5. Keep the selected option as "Commit directly". You don't need to create a new branch for every change.
+6. Press "Commit changes".
+
+### Finalising the change
+
+1. On the "Conversation" tab of your Pull Request, click "Merge pull request".
```
Comment (StevenLeighton21) [inline guides/contributing_to_the_handbook.md]: Suggest a note on PR etiquette? As simple as `When your proposal has approval, click....`

Comment (zuren) [reply guides/contributing_to_the_handbook.md]: Nice idea.

**Comment by yaseminercan** [comment]
Great, thank you Richard this is super helpful! I'll use it next time I'm adding a page and go through it step by step.

File: guides/contributing_to_the_handbook.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
+# Contributing to the Handbook
+
+This guide aims to make it easier to submit changes to the Handbook without using `git` or a text editor.
+
+## Adding new pages in Github
+
+### Creating the page
```
Comment (ccp92) [inline guides/contributing_to_the_handbook.md]: Maybe could do with something around editing an existing file as well?

Comment (zuren) [reply guides/contributing_to_the_handbook.md]: Out of scope! Good point though.

**Comment by zuren** [comment]
@lukemorton done.

**Pull request merged**